src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,13 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllCommentsQuery = session.createQuery(
+                    "FROM Movie", Movie.class
+            );
```

Comment:
in my opinion, it is better to write in one line

Reply:
The same as above